### PR TITLE
#54: replaces AttrInjection() method with AttrsInjection()

### PR DESCRIPTION
**Pull request by SGorvalds**
Hello again, @egoholic!
I've successfully fixed the next issue #54. Check this out in your spare time, please.

### CODE REVIEW

File: go2html.go [hunk 1]
```diff
@@ -100,6 +100,12 @@ func (t *Template) Populate(rawReplacements map[string]interface{}) string {
 				result := fragment.template.Populate(nestedReplacement)
 				sb.WriteString(result)
 			}
+		case attrsInjection:
+			rawRepl, _ := rawReplacements[string(fragment)]
+			repl, _ := rawRepl.(map[string]string)
+			for key, val := range repl {
+				sb.WriteString(fmt.Sprintf("%v=%v", key, val))
```
Comment (egoholic) [inline go2html.go:107]: attribute value should be within `"..."`

File: go2html.go [hunk 2]
```diff
@@ -100,6 +100,12 @@ func (t *Template) Populate(rawReplacements map[string]interface{}) string {
 				result := fragment.template.Populate(nestedReplacement)
 				sb.WriteString(result)
 			}
+		case attrsInjection:
+			rawRepl, _ := rawReplacements[string(fragment)]
+			repl, _ := rawRepl.(map[string]string)
```
Comment (egoholic) [inline go2html.go:105]: let's panic for now if we can't assert the type of raw replacements

File: go2html_test.go [hunk 3]
```diff
@@ -103,10 +103,10 @@ var _ = Describe("go2html", func() {
 							map[string]interface{}{
 								"text1":             "Inserted text here.",
 								"paragraph1":        "Inserted <b>paragraph1</b> text.",
-								"div1-attr":         "title=\"Some title\"",
+								"div1-attr":         map[string]string{"title": "\"Some title\""},
```
Comment (egoholic) [inline go2html_test.go:106]: here we should avoid formatting for values (I mean `\"...\"`), the value string should be an exact value that will be placed later within `<attr>="<value>"`

**Review by egoholic — CHANGES_REQUESTED** [review]

File: elem_cfgr.go [hunk 4]
```diff
@@ -15,10 +15,8 @@ type (
 	}
 )
 
-func (ecp *ElemCfgr) AttrInjection(key string) {
-	fragment := injection{
-		key: key,
-	}
+func (ecp *ElemCfgr) AttrsInjection(key string) {
```
Comment (egoholic) [inline elem_cfgr.go:18]: rename `ATTRIBUTE_INJECTION_NODE_KIND` to `ATTRIBUTES_INJECTION_NODE_KIND` and you will see all the  functions/methods that require some updates: `node.String()` and `node.New()`

File: go2html.go (hunk 1 above)
Comment (egoholic) [reply go2html.go:107]: also replace `%v` with `%s` formatters

Comment (SGorvalds) [reply go2html.go:107]: I agree, that's a more neat way of building the string. I've fixed that and implemented corresponding requested changes below.

File: go2html.go (hunk 2 above)
Comment (SGorvalds) [reply go2html.go:105]: Fixed, as said above in the first changes request.

File: go2html_test.go (hunk 3 above)
Comment (SGorvalds) [reply go2html_test.go:106]: Fixed, as said above in the first changes request.

File: elem_cfgr.go (hunk 4 above)
Comment (SGorvalds) [reply elem_cfgr.go:18]: Done! But FYI, `String()` and `New()` are not methods of the `Node` structure. They are separate functions.

File: go2html.go (hunk 2 above)
Comment (SGorvalds) [reply go2html.go:105]: Done!

**Comment by egoholic** [comment]
great work! 🥇

**Pull request merged**